src/improvements/tasks/sep/000-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -681,6 +683,7 @@ For each contract listed in the state diff, please verify that no contracts or s
                        The Raw Slot can be derived from:
                         - <i>cast index uint32 1 101</i>
 
+  ----- NOT IN TRACE -----
```

Comment:
what do these `NOT IN TRACE` lines mean? How should a signer interpret them?

Reply:
The storage changes between two NOT IN TRACE tags where not in my trace. The info was more for you than the signers. I've removed those storage changes from my branch so that it is clearer